src/AdminGuesser.js

```diff
@@ -24,25 +28,19 @@ export const render = (
     return <div>Error while reading the API schema</div>;
   }
 
-  const childrenAsArray = React.Children.toArray(children);
-  data.resources
-    .filter(
-      resource =>
-        !resource.deprecated &&
-        !blacklist.includes(resource.name) &&
-        existsAsChild(children)(resource.name),
-    )
-    .forEach(resource =>
-      childrenAsArray.push(
-        <ResourceGuesser name={resource.name} key={resource.name} />,
-      ),
-    );
+  if (!children) {
+    children = data.resources
+      .filter(r => !r.deprecated)
+      .map(r => <ResourceGuesser name={r.name} key={r.name} />);
+  }
```

Comment:
Is it on purpose to not handle a control boolean like in IntrospectQuery to include deprecated resources ?

Reply:
Control props added. To be documented.